frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -1526,6 +1531,11 @@ var AppRouter = Backbone.Router.extend({
                     path: [countlyGlobal.cdn + 'localization/min/'],
                     mode: 'map',
                     callback: function() {
+                        if (countlyGlobal.company) {
+                            for (var key in jQuery.i18n.map) {
+                                jQuery.i18n.map[key] = jQuery.i18n.map[key].replace("Countly", countlyGlobal.company);
+                            }
+                        }
```

Comment:
Can you also recheck prelogin pages if they are handled, they should be handled in countly/frontend/express/public/javascripts/prelogin.js